src/shared/data-fd-util.c

```diff
@@ -343,3 +348,50 @@ int copy_data_fd(int fd) {
 
         return fd_reopen(tmp_fd, O_RDONLY|O_CLOEXEC);
 }
+
+int memfd_clone_fd(int fd, const char *name, int mode) {
+        _cleanup_close_ int mfd = -EBADF;
```

Comment:
It doesn't matter greatly, but we always use -1 for fd initialization. Even in this function, -1 is used below.

Reply:
oh, see discussion on #25513. LXC codebases do it like this, and I think it makes sense to just say -EBADF is the "invalid" value for fds, rather than the usual -1?

So we started merging code with -EBADF instead of -1 as "bad fd" indicator, and I think it's a scheme that makes a ton of sense and we should follow for new code everywhere.